docs/source/queries/paged.md

```diff
@@ -80,7 +80,7 @@ On a `Query`:
 use scylla::query::Query;
 
 let mut query: Query = Query::new("SELECT a, b FROM ks.t");
-query.set_page_size(16.try_into().unwrap());
+query.set_page_size(16);
 
 let _ = session.query_iter(query, &[]).await?; // ...
 # Ok(())
```

Comment:
I'm wondering why this even passed CI. Shouldn't those examples fail to compile before this change?

Reply:
There are blanket impls ;)
`impl TryFrom<T> for T { ... }`